package.json

```diff
@@ -69,6 +68,8 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
+    "@formatjs/intl-listformat": "^7.5.7",
```

Comment:
I'm not sure this should be a dev dependency? I think it needs to be a dependency as it's needed in the built version?

Reply:
Ah you're totally right - thanks for catching!